script/run-autoware-traffic-light.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+export VEHICLE_NAME="${1:-v1}"
+export ZENOH_CARLA_IP_PORT="${2:-'127.0.0.1:7447'}"
+export ZENOH_FMS_IP_PORT="${3:-'127.0.0.1:7887'}"
+
+# Add default port for Zenoh IP in Carla and FMS
+if [[ ${ZENOH_CARLA_IP_PORT} != *":"* ]]; then
+    export ZENOH_CARLA_IP_PORT="${ZENOH_CARLA_IP_PORT}:7447"
+fi
+if [[ ${ZENOH_FMS_IP_PORT} != *":"* ]]; then
+    export ZENOH_FMS_IP_PORT="${ZENOH_FMS_IP_PORT}:7887"
+fi
+
+# Log folder
+LOG_PATH=autoware_log/`date '+%Y-%m-%d_%H:%M:%S'`/
+mkdir -p ${LOG_PATH}
+
+# Run the program
+parallel --verbose --lb ::: \
+    "ros2 launch autoware_carla_launch autoware_zenoh.launch.xml is_simulation:=false use_traffic_light_recognition:=true \
```

Comment:
I forgot why we need `is_simulation:=false`. Could you please add the comment here?